mongodb/trigger/eventlistener/descriptor.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "mongodb-eventlistener",
+    "type": "flogo:trigger",
+    "title": "mongodb-eventlistener",
+    "version": "1.0.0",
```

Comment:
Are we confident to use the the "1.0.0" version given that this is the first version of this code is OSS?

Reply:
Since this connector will also be released in FE using the same code as that of OSS, I really cannot have two different versions; one for FE and one for OSS.
Sriharsha from FE team confirmed that we cannot have one version for OSS and use decorator pattern that FE team has come up with to have another version for FE contribution. If there is no harm is sticking to 1.0.0, it would be great..